### DISCUSSION TITLE
Print labels using pre-determined dimensions?

### DISCUSSION THREAD

**Discussion by jkleinkopf** [Symbiota/community-board#417] [Wish List]
Hi, I just wanted to put in a request for a feature to print specimen labels with pre-determined margins and dimensions. We want to  print labels on Avery label sheets, to make easy-to-use labels for tissue samples held in coin envelopes. Specifically, we are interested in printing specimen labels on 2" x 2-5/8" Avery sheets (https://www.avery.com/blank/labels/94235), but it may be helpful to us and others if we are able to print on other pre-determined label sheets.

Thanks!
Joseph

**Comment by STHerbariumLSU** [comment]
Similarly with annotation labels, it would be nice to be able to scale the footer.  The font size is large for our herbarium, so I always have to do a find and replace to scale it down.  Is it possible to have more functions here?

**Comment by themerekat** [comment]
@jkleinkopf and @STHerbariumLSU, here's a little secret: you can do a _lot_ more customization of labels if you dive into customizing your own CSS file. The default parameters for printed labels are defined in this file: https://github.com/BioKIC/Symbiota/blob/master/css/v202209/symbiota/collections/reports/labelhelpers.css
You can create your own version of this file and invoke it here in the label profile creation tools:
![image](https://github.com/BioKIC/symbiota-docs/assets/16688784/add6e0ff-1435-4e47-8f7b-06e8fabcdd7d)

You would have to send us that file so we can load it into the portal's css directory, but technically, you can get as custom as you want with one of these!

**Comment by tcbarry** [comment]
[themerekat](https://github.com/themerekat) I am very interested in a learning more about customizing our own CSS file so we can customize our labels without numerous edits to the exported Word document from the "Label Printing" module in Symbiota. I tried to follow the GitHub link you provided but got a "404 page not found" error. 

Thank you, Teri

**Comment by themerekat** [comment]
@tcbarry , sorry, that file has moved! You can find it here now: https://github.com/BioKIC/Symbiota/blob/master/css/symbiota/collections/reports/labelhelpers.css

**Comment by tcbarry** [comment]
Thank you.

I would like instruction or a future workshop for this. Does this require extensive knowledge of code?

Teri Barry
Collections Manager
UC Davis Center for Plant Diversity Herbarium
***@***.***
(530)752-1091

**Comment by themerekat** [comment]
@tcbarry it does require an understanding of CSS, which is code-ish. We'll keep that in mind!
